src/_guides/language/concurrency/index.md

```diff
@@ -469,7 +506,6 @@ TODO:
 * Maybe:
   * Add a new macro & style for flutter notes?
   * Add the following text somewhere in this page (or in the FAQ):
-    * Sometimes Dart is called a _single-threaded language._
```

Comment:
Are the rest of these TODOs still relevant?

Reply:
No, not for me at least. I'll incorporate the "Dart code executes..." line then remove the whole comment in my next edit.

Reply:
Removing the TODOs and maybes seems good to me! The important ones are fixed or tracked already.